iChef-WebApi/src/main/java/com/kitchen/iChef/Repository/UserRepository.java

```diff
@@ -0,0 +1,53 @@
+package com.kitchen.iChef.Repository;
+
+import com.kitchen.iChef.Domain.AppUser;
+import com.kitchen.iChef.Repository.Interfaces.ICrudRepository;
+import com.kitchen.iChef.Repository.Interfaces.IUserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+@Profile({"dev", "default"})
+public class UserRepository implements ICrudRepository<AppUser, String> {
```

Comment:
Shouldn't this implement IUserRepository?

Reply:
IUserRepository is using springframework methods while this repository is for our own methods

Reply:
What are you talking about? IUserRepository is an interface that extends the ICrudRepository interface and sets the entity type and the id type to AppUser and string.